src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1100,23 +1100,34 @@ protected function send_plugin_theme_email( $type, $successful_updates, $failed_
 				$body[] = __( 'These plugins failed to update:' );
 
 				foreach ( $failed_updates['plugin'] as $item ) {
+					$body_message = '';
+					$item_url     = '';
+
+					if ( ! empty( $item->item->url ) ) {
+						$item_url = ' : ' . esc_url( $item->item->url );
+					}
+
 					if ( $item->item->current_version ) {
-						$body[] = sprintf(
-							/* translators: 1: Plugin name, 2: Current version number, 3: New version number. */
-							__( '- %1$s (from version %2$s to %3$s)' ),
+						$body_message .= sprintf(
+							/* translators: 1: Plugin name, 2: Current version number, 3: New version number, 4: Plugin URL. */
+							__( '- %1$s (from version %2$s to %3$s)%4$s' ),
```

Comment:
```suggestion
							__( '- %1$s (from version %2$s to %3$s) %4$s' ),
```

🔢 This applies to others, they'll need a space too.

Reply:
The space is included in `$item_url`:

```php
$item_url = ' : ' . esc_url( $item->item->url );
```

Otherwise, with an empty `$item_url`, there would be a trailing space.

Reply:
Thanks.

I think this might be problematic for i18n as it assumes the sentence works with `: http://example.com/colins-plugin` and I don't think that's reliably the case.

Unfortunately this involves doubling the number of strings again so I'll defer to people who know much better than I for confirmation. @SergeyBiryukov @audrasjb do you have thoughts here?